GAE/src/org/waterforpeople/mapping/dataexport/RawDataSpreadsheetImporter.java

```diff
@@ -84,27 +85,28 @@
     public static final String COLLECTION_DATE_COLUMN_KEY = "collectionDate";
     public static final String SUBMITTER_COLUMN_KEY = "submitterName";
     public static final String DURATION_COLUMN_KEY = "surveyalTime";
-    
+
     public static final String METADATA_HEADER = "Metadata";
     public static final String OTHER_SUFFIX = "--OTHER--";
     public static final String NEW_DATA_PATTERN = "^[Nn]ew-\\d+"; // new- or New- followed by one or more digits
-    
+
 
     /**
      * opens a file input stream using the file passed in and tries to return the first worksheet in
-     * that file. 
+     * that file.
      * Also called from uploader.clj.
      *
      * @param file
      * @return
      * @throws Exception
      */
-    public Sheet getDataSheet(File file) throws Exception {
+    public Sheet getDataSheet(File file) {
         stream = new PushbackInputStream(new FileInputStream(file));
         Workbook wb = null;
         try {
             wb = WorkbookFactory.create(stream);
         } catch (Exception e) {
+            Log.error("Workbook creation exception:" + e);
```

Comment:
It seems this have added a new import: org.jfree.util.Log, why not use the same log object as line 204? log.error("Workbook creation exception", e); ? Is there a reason behind using 2 different ways of logging errors?